app/views/spotlight/sir_trevor/blocks/_text_block.html.erb

```diff
@@ -1,3 +1,3 @@
 <div class="st__content-block st__content-block--text">
-	<%= sir_trevor_markdown text_block.text %>
+  <%= sir_trevor_format text_block.text, format: text_block.format%>
```

Comment:
Did we intentionally decide to use sir_trevor_markdown only, and if so, should the change here on line 2 be reverted?